src/test/java/com/stripe/functional/PersonTest.java

```diff
@@ -0,0 +1,120 @@
+package com.stripe.functional;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import com.stripe.BaseStripeTest;
+import com.stripe.exception.StripeException;
+import com.stripe.model.Account;
+import com.stripe.model.Person;
+import com.stripe.model.PersonCollection;
+import com.stripe.net.ApiResource;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Test;
+
+public class PersonTest extends BaseStripeTest {
+  public static final String ACCOUNT_ID = "cus_123";
+  public static final String PERSON_ID = "person_123";
+
+  private PersonCollection getPersonCollectionFixture(Account account)
+      throws IOException, StripeException {
+    PersonCollection persons = account.persons();
+    resetNetworkSpy();
+
+    return persons;
+  }
+
+  @Test
+  public void testCreate() throws IOException, StripeException {
+    final Account account = Account.retrieve(ACCOUNT_ID, null);
+    PersonCollection personsTmp = getPersonCollectionFixture(account);
```

Comment:
You might as well move the call to `Account.retrieve()` inside `getPersonCollectionFixture()` and remove the argument.

Reply:
Also you can declare `personsTmp` as `final`.

Reply:
> You might as well move the call to Account.retrieve() inside getPersonCollectionFixture() and remove the argument.

That won't work since I need the account id to verify the path. I could hardcode it but we've had issues before where the resource id in the path is not the one you asked for so it felt cleaner. Do you disagree?

> Also you can declare personsTmp as final.

ack